packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -179,6 +185,14 @@ export const SelectTree = function SelectTree( {
 					setInputValue( '' );
 					recalculateInputValue();
 				}
+			} else if (
+				( event.key === 'ArrowLeft' || event.key === 'Backspace' ) &&
```

Comment:
Should the "Backspace" have the same behavior as the "ArrowLeft" when the input is focused?

Reply:
Yes. When nothing is selected, and the other conditions also match, we should focus the last token.